example/android/app/src/main/AndroidManifest.xml

```diff
@@ -62,5 +62,11 @@
           </intent-filter>
       </service>
 
+        <service
+            android:name="com.bitmovin.player.reactnative.services.BackgroundPlaybackService"
+            android:foregroundServiceType="mediaPlayback"
+            android:exported="true">
+        </service>
+
```

Comment:
**Note**
This is necessary also for correctly updating the playerview after the app has been minimized. In fact, lifecycle changes seems to not be enough without this (the whole playerview will be black).